.github/workflows/e2e-test-k3s.yml

```diff
@@ -5,17 +5,21 @@
 #    - Checkout the repository code.
 #    - Cache the ~/.cache/go-build and ~/go/pkg/mod directories to improve performance.
 #    - Set up a docker registry and k3s cluster using a script called k3s-setup.sh.
-#    - Set the pull request number as an environment variable.
-#    - Run Go tests located in the testing/go directory, ensuring that the correct version of Go is used and the Go modules are tidied up before the tests are run.
-#
-#  Overall, this action ensures that the code changes made in the pull request do not break any end-to-end tests on a k3s cluster.
+#    - Set the pull request number, OCI module image name as environment variables.
+#    - Wait until the registry contains OCI module image for current PR (pushed by prow-job)
+#    - Run test script located in the testing directory
 
+name: Run E2E tests on k3s cluster
 
-name: e2e-test-k3s
+env:
+  BTP_OPERATOR_DEV_REPO: europe-docker.pkg.dev/kyma-project/dev/unsigned/component-descriptors/kyma.project.io/module/btp-operator
+  BTP_MANAGER_DEV_REPO: europe-docker.pkg.dev/kyma-project/dev/btp-manager
+  TAG_PREFIX: 0.0.0
```

Comment:
`0.0.0` or `v0.0.0`?

Reply:
Yes. Now I am confused and we need to reconsider naming afer latest release with v0.2.3 and 0.2.3 values in different repos.

Reply:
https://sap-btp.slack.com/archives/G01L562297B/p1677762333871759?thread_ts=1677753000.118519&cid=G01L562297B

Reply:
we need to keep the pattern as `0.0.0`, there were changes in the module creating command which adds `v` to the version.